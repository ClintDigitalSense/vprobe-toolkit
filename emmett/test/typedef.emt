typedef int booze;
booze bar;

VMM1Hz {
   bar = bar + 1;
   printf("bar: %d\n");
}
