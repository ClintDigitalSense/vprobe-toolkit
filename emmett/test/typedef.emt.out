(version 0.2)
(definteger bar)

 ;;;;;
(vprobe VMM1Hz ( do  ( setint bar    (+ bar  1 ))( printf    "bar: %d\n" )))